nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -474,6 +474,11 @@ in
               type = types.path;
               description = lib.mdDoc "The mount point of the directory inside the virtual machine";
             };
+            options.securityModel = mkOption {
+              type = types.enum [ "passthrough" "mapped" "mapped-xattr" "mapped-file" "none" ];
+              default = "none";
```

Comment:
It would be a good idea to change this to `mapped-xattr`. This is what qemu recommends and shouldn't cause issues, though it would require some testing.

Reply:
I agree that it would be sensible to use the recommended model as default. However with the proposed change it will change the security model for the nix-store as well. I'll look into it! 
Thanks!

Reply:
>  it will change the security model for the nix-store as well. I'll look into it!

Yeah, that's why we should be careful. I'll do some testing as well.